source/guide/3.2/plugin-softlayer.md

```diff
@@ -0,0 +1,468 @@
+---
+layout: bt_wiki
+title: SoftLayer Plugin
+category: Plugins
+publish: true
+abstract: Cloudify SoftLayer plugin description and configuration
+pageord: 600
+---
+
+
+{%summary%}
+This section describes how to use SoftLayer based cloud infrastructure in your services and applications.
+For more information about SoftLayer, please refer to: [http://www.softlayer.com/](http://www.softlayer.com/).
+{%endsummary%}
+
+
+# Types
+
+## cloudify.softlayer.nodes.VirtualServer
+
+
+**Derived From:** [cloudify.nodes.Compute](reference-types.html)
+
+
+**Properties:**
+
+  * **Required Properties:**
+    * `location` The data center.
```

Comment:
that the "The data center" mean? Is it a name?
Maybe reference to softlayer's documentation?